duckduckgo/duckduckgo-privacy-extension
# UI updates: num trackers ui to Safari branch

**Reviewer:**
@jdorweiler 

**CC:**
@MariagraziaAlastra @andrey-p 

## Description:
Ports UI changes that were merged to develop for Chrome/FF over to Safari.


## Steps to test this PR:
<!-- List steps to test it manually 
1. Build branch
2. Confirm trackers subview hero UI updates.

![safari ui updates](https://user-images.githubusercontent.com/1278969/35893280-1d24d212-0b62-11e8-94d5-549b145438d3.png)


## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
